crypto/asn1/a_time.c

```diff
@@ -130,7 +130,7 @@ int ASN1_TIME_set_string(ASN1_TIME *s, const char *str)
     return 1;
 }
 
-static int asn1_time_to_tm(struct tm *tm, const ASN1_TIME *t)
+int ASN1_TIME_to_tm(struct tm *tm, const ASN1_TIME *t)
```

Comment:
I will note that ASN1_TIME_print() is an anomaly (not suggesting this changes the need to reverse the arguments).